openapi/gitlab/client.bal

```diff
@@ -0,0 +1,261 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import  ballerina/http;
+import  ballerina/url;
+import  ballerina/lang.'string;
+
+public type ApiKeysConfig record {
```

Comment:
Can we manually add a doc comment explaining the api key / value pairs that need to be added? Follow [this](https://github.com/ballerina-platform/ballerinax-openapi-connectors/blob/81573c7e1cde84f87ae46718819c68b95f8c920e/openapi/newsapi/client.bal#L21) as a sample.

Reply:
Added